README.rst

```diff
@@ -48,21 +50,47 @@ Install ``scikit-hep`` like any other Python package:
 
 .. code-block:: bash
 
-   pip install scikit-hep
+   python -m pip install scikit-hep
 
-or similar (use ``virtualenv``, etc. if you wish).
+or similar (use ``venv``, etc. if you wish).
 
-Strict dependencies
-...................
+Package version and dependencies
+................................
 
 Please check the ``setup.py`` and ``requirements.txt`` files for the list
 of Python versions supported and the list of Scikit-HEP project packages
 and dependencies included, respectively.
 
-Documentation
-.............
-
-The project documentation is hosted on the `Scikit-HEP website`_.
-
-
-.. _Scikit-HEP website: http://scikit-hep.org/
```

Comment:
Did we lose this nice pointer to the website?

Reply:
No, see the top of the README - "The `Scikit-HEP project <http://scikit-hep.org/>`_ ...". In short, I simplified.

Reply:
Great, thanks!